esmf_regrid/_esmf_sdo.py

```diff
@@ -35,6 +36,11 @@ def shape(self):
         """Return shape."""
         return self._shape
 
+    @property
+    def _extended_shape(self):
```

Comment:
I suppose that fits better into the naming scheme. I guess I was avoiding them because GridInfo didn't do any refining, bu I think that maybe another word here confuses the matter.

Reply:
>I guess I was avoiding them because GridInfo didn't do any refining

For me it's clear enough given that `_refined_shape` is the same as `_shape` in that case.

Reply:
c473e0dee2711a58581ac78fc0bd89fad58c6c2d